share/getter.go

```diff
@@ -73,15 +73,15 @@ func (ns NamespacedShares) Verify(root *Root, namespace Namespace) error {
 
 	for i, row := range ns {
 		// verify row data against row hash from original root
-		if !row.verify(originalRoots[i], namespace) {
+		if !row.Verify(originalRoots[i], namespace) {
```

Comment:
A minor change from local changes. It makes sure we don't segfault when proof is malformed.
```suggestion
		if row.Proof == nil && row.Shares == nil {
			return fmt.Errorf("row verification failed: no proofs and shares")
		}
		if !row.Verify(originalRoots[i], namespace) {
```